### PR TITLE
Update to latest nodejs & typescript bazel rules

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,12 +53,14 @@ jobs:
...
 
+      - run: sudo cp .circleci/bazel.rc /etc/bazel.bazelrc
+
       # Helpful for debugging, so you can check you have the same bazel version when you need
       # to reproduce a failure.
       - run: bazel info release
 
       # Install the dependencies from NPM
-      - run: bazel run --config=ci @yarn//:yarn install
+      - run: bazel run --config=ci @nodejs//:yarn install
```
Comment (alexeagle) [inline .circleci/config.yml]: like the rules_typescript PR - remove `--config=ci` everywhere and remove the `build:ci` lines from `tools/bazel.rc`

File: src/BUILD.bazel [hunk 2]
```diff
@@ -40,6 +34,15 @@ rollup_bundle(
     deps = ["//src"],
 )
 
+# Needed because the prodserver only loads static files that appear under this
```
Comment (alexeagle) [inline src/BUILD.bazel]: is this really true? we completely control the args to http-server, is it possible to get it to serve zone.js from a different directory?

File: .circleci/config.yml (hunk 1 above)
Comment (gregmagolan) [reply .circleci/config.yml]: 👍

File: src/BUILD.bazel (hunk 2 above)
Comment (gregmagolan) [reply src/BUILD.bazel]: looks like it is. http-server doesn't support multiple root dirs and couldn't find any other node http server that does

**Comment by gregmagolan** [comment]
Now waiting on a 0.14.1 rules_typescript release (0.14.0 was broken by a change in the yarn registery that changed the path to karma-sourcemap-loader)

**Comment by gregmagolan** [comment]
Updated to rules_typescript 0.15.0 now

**Pull request merged**